docs/api/api.md

```diff
@@ -0,0 +1,254 @@
+# API
+
+Please have a look at the documentation and examples of the interactive Swagger interface: [https://oqt.ohsome.org/api/docs](https://oqt.ohsome.org/api/docs)
+
+
+## Request examples using Bash and CURL
+
+Using the above mentioned Swagger interface one can create valid `curl` requests using an interactive interface.
+
+In the following sections examples for a GET and POST request for an Indicator of a region pre-definied by OQT are given.
+
+### Request an Indicator for a pre-definied region
+
+#### GET request
+
+```bash
+curl --request GET "https://oqt.ohsome.org/api/indicator?name=GhsPopComparisonBuildings&layerName=building_count&dataset=regions&featureId=3"
+```
+
+#### POST request
+
+```bash
+curl --request POST \
+    --header "Content-Type: application/json" \
+    --data '{"name": "GhsPopComparisonBuildings", "layerName": "building_count", "dataset": "regions", "featureId": 3}' \
+    "https://oqt.ohsome.org/api/indicator"
+```
+
+> Tip: Use Python to pretty-print the response JSON -> `curl [...] | python -m json.tool`
+
+
+### Request an Indicator for custom bounding polygon(s)
+
+Since GeoJSON strings are quite big it is recommended that POST requests are used.
+
+#### POST request
+
+Let's write down the request parameters in a `JSON` file.
+Note that the bpolys parameters needs to be a valid GeoJSON object.
+
+```bash
+{
+  "name": "GhsPopComparisonBuildings",
+  "layerName": "building_count",
+  "bpolys": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          8.674092292785645,
+          49.40427147224242
+        ],
+        [
+          8.695850372314453,
+          49.40427147224242
+        ],
+        [
+          8.695850372314453,
+          49.415552187316095
+        ],
+        [
+          8.674092292785645,
+          49.415552187316095
+        ],
+        [
+          8.674092292785645,
+          49.40427147224242
+        ]
+      ]
+    ]
+  }
+}
+```
+
+Now that a `JSON` file exists on disk one can post it with `curl` using following syntax `curl -X POST -H "Content-Type: application/json" -d @FILENAME DESTINATION`.
+E.g.
+
+```bash
+curl --request POST \
+    --header "Content-Type: application/json" \
+    --data @data.json \
+    "https://oqt.ohsome.org/api/indicator"
+```
+
+In reality one wants to use a tool to URL encode a valid GeoJSON file for requests. In the next section Python is used to archive this.
```

Comment:
* `curl` is capable of doing URL encoding, see [`--data-urlencode`](https://manpages.debian.org/bullseye/curl/curl.1.en.html#data-urlencode)
* Just a suggestion: prevent the phrase _in reality_
```suggestion
Most advanced users use a tool to include a valid GeoJSON file into their requests. The next section shows an example that uses Python to achieve this.
```